docs/metric-interpretation-guide/README.md

```diff
@@ -0,0 +1,141 @@
+to manipulate the scope or quality of the suite. Test suite size should correspond to the state of
+the product. For example, a product under active development should show a gradual increase in the
+size of its test suite, while a product in maintenance should exhibit more stable trends.
+
+**Success Rates**
+
+Success rates provide a quick indication of the test suite's health. A low success rate or high
+failure rate signals potential quality issues, either in the product or within the test suite
+itself. This metric can be tracked on both a test-by-test basis and for the entire test suite.
+
+**Average Success Rates**
+
+To avoid noise from isolated failures and spot trends more easily, it's helpful to calculate
+averaged success rates over time. This allows teams to act early if trends toward failure begin to
+emerge, preventing the test suite from becoming unreliable or mistrusted. Success rate averages are
+calculated as:
+
+```text
+100 x (Successful Runs / (All Runs - Cancelled Runs))
+```
+
+These averages can be calculated over 30-day, 60-day, and 90-day periods, with the 90-day trend
+being preferred. Average success rates are typically interpreted as follows, though teams may adjust
+thresholds based on their specific needs:
+
+| Threshold | Interpretation                                                     |
+|-----------|--------------------------------------------------------------------|
+| \>= 95%   | Healthy - Tests pass the majority of the time                      |
+| 90% - 95% | Caution - Tests show signs of instability, requiring investigation |
+| < 90%     | Critical - Tests are faulty and need intervention                  |
+
+## Time Measurements
+
+_Supported Test Frameworks:_ [jest][1], [mocha][2], [nextest][3], [playwright][4], [pytest][5]\
+_Supported CI:_ [CircleCI][7]
+
+Time measurements track how long it takes for tests to run. Ideally, these times should be
+proportional to the size of the test suite and remain stable over time. Significant increases or
+variations in execution time may indicate performance issues or inefficiencies within the test
+suite. Monitoring execution times allows teams to identify and address bottlenecks to keep test
+suites efficient.
+
+**Run Time**
+
+The cumulative time of all test runs in a suite.
+
+**Execution Time**
+
+The total time taken for the test suite to execute. If tests are not run in parallel, the execution
+time should match the run time.
+
+**Job Time**
+
+The time taken for the test job to complete in CI. Job time thresholds are typically interpreted as
+follows:
+
+| Threshold | Interpretation                                             |
+|-----------|------------------------------------------------------------|
+| \> 10m    | Slow - The test suite may require optimization             |
+| <= 10m    | Fast - The test suite runs within an acceptable time frame |
+
+## Coverage Metrics
+
+_Supported Coverage Frameworks:_ [pytest-cov][8], [llvm-cov][9]
```

Comment:
I think this should be named `Support Coverage Tools` rather than `Frameworks` as for Frameworks we have `jest, mocha, tap, intern, pytest etc`

Reply:
Which FxA suite uses intern? We aren't explicitly covering it in these scripts, but maybe we should.